content/tutorial-air-quality-analysis.md

```diff
@@ -0,0 +1,251 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Analysis of air pollution levels before and after lockdown in Delhi, India
+
+
+## What you'll do
+
+Calculate Air Quality Indices (AQI) and perform paired t-test on them.
+
+## What you'll learn
+
+- You'll compute moving averages
+
+- You'll calculate Air Quality Index (AQI)
+
+- You'll perform a paired t-test and find the t and p values
+
+- You'll learn to interpret these values
```

Comment:
I feel like these are more "What you'll do" and only the last bullet actually talks about what the reader will learn. Maybe this can be rephrased.